data/campaigns/emp04.wmf/scripting/mission_thread.lua

```diff
@@ -31,16 +31,15 @@ function farm_plans()
    local f = map:get_field(47, 10)
    local farmclick = false
    local count = 0
-   local o1 = add_campaign_objective(obj_click_farmbuilding)
-   o1.done = true
+   local o1 = nil
```

Comment:
Good catch :) 
When you set `o1.done = true` below in line 47 you probably need to check whether `o1` is not `nil` though.

Reply:
implemented. Always easy to find your own wrong habits. (that was the case I mentioned before)